src/main/java/org/icgc/argo/program_service/exception/ExceptionHandlers.java

```diff
@@ -63,4 +64,19 @@ public ResponseEntity<Object> handleNotFoundException(
         new HttpHeaders(),
         NOT_FOUND);
   }
+
+  @ExceptionHandler(BadRequestException.class)
+  public ResponseEntity<Object> handleForbiddenException(
+      HttpServletRequest req, BadRequestException ex) {

```

Comment:
Bad Request and Forbidden are different error types... we should have handling for auth failures that is different from invalid request parameter errors. Can you implement both separately?

Reply:
modified the exception handler method to "handleBadRequestException" as there is already another method to handle ForBiddenException